recipe/meta.yaml

```
@@ -32,13 +34,28 @@ requirements:
 test:
   imports:
     - requests
+  source_files:
+    - tests
+    - pyproject.toml
   requires:
     - pip
-    - conda  # [not py==313]
+    - conda
+    - pytest >=6.2
```

https://github.com/psf/requests/blob/v2.32.4/setup.py#L48
```suggestion
    - pytest >=3
```

---

For some reason, the maintainers updated it to 6.2 in pyproject.toml https://github.com/psf/requests/blame/v2.32.4/pyproject.toml#L9